x11-misc/xwit/xwit-3.4_p15.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit toolchain-funcs
+
+DESCRIPTION="A collection of simple routines to call some of those X11 functions"
+HOMEPAGE="https://tracker.debian.org/pkg/xwit"
+SRC_URI="
+	mirror://debian/pool/main/x/${PN}/${PN}_${PV/_p*/}.orig.tar.gz
+	mirror://debian/pool/main/x/${PN}/${PN}_${PV/_p/-}.debian.tar.gz
+"
+
+LICENSE="public-domain HPND"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="x11-libs/libX11"
+DEPEND="${RDEPEND}
+	x11-proto/xproto"
+
+S="${WORKDIR}"/"${P/_p*/}".orig
```

Comment:
either drop all quotes (which is fine for a variable assignment) or put all of it quotes, but don't ever define a variable by using two quotes segments